src/TurbKineticEnergyEquationSystem.C

```diff
@@ -337,6 +341,7 @@ TurbKineticEnergyEquationSystem::register_interior_algorithm(
       break;
     }
     case SST:
+    case SST_BLT:
```

Comment:
```suggestion
```

Reply:
@psakievich Yes, I noticed that soon after I pushed my changes. I took all references to SST_BLT out. Thanks.